src/clr-angular/data/datagrid/datagrid-filter.ts

```diff
@@ -25,8 +25,9 @@ import { ClrCommonStrings } from '../../utils/i18n/common-strings.interface';
   providers: [{ provide: CustomFilter, useExisting: ClrDatagridFilter }],
   template: `
         <button #anchor class="datagrid-filter-toggle" (click)="toggle()"
-            [class.datagrid-filter-open]="open" [class.datagrid-filtered]="active"
-            type="button"></button>
+                [class.datagrid-filter-open]="open" [class.datagrid-filtered]="active">
+            <clr-icon [attr.shape]="active ? 'filter-grid-circle': 'filter-grid'" class="is-solid"></clr-icon>
+        </button>
```

Comment:
Was the `type="button"` supposed to be removed?

Reply:
nope. I forgot to put it back once i got it working. Will add it.